app/js/components/common/Header.jsx

```diff
@@ -60,7 +60,7 @@ export default class Header extends Component {
       });
     });
 
-    this.fetchCurrentUserLocation('/v1/appui/session').then((response) => {
+    this.fetchCurrentUserLocation('/v1/location ').then((response) => {
```

Comment:
I am curious to know why you have 2 methods in this component making api request with the same endpoint.

Reply:
Secondly from the ticket description, i understand the console error is coming from this endpoint `/v1/appui/session`. So i just want to know if you made some research to check if the endpoint was implemented correctly rather than getting rid of the `this.fetchCurrentUserLocation` method entirely.